Improved the css

khushiiagrawal opened the pull request:
This pull request includes changes to the `public/index.html` and `public/styles.css` files to improve the user interface and styling of the Indian City Guessing Game. The most important changes include adding placeholders and IDs to form elements, and updating styles for better user experience.

Improvements to user interface:

* [`public/index.html`](diffhunk://#diff-c2cc24bc9001b11b6add48a4cd8f893d5d6c6e4d1bd254158bd14ab997f552cdL16-R23): Added a placeholder to the `player-name` input field and assigned IDs to the label and button elements to improve accessibility and user experience.

Styling updates:

* [`public/styles.css`](diffhunk://#diff-c0957aceb7c1368cd9919f48a72240ae31b22112283b1bd4d38c66a55ed584b9R35-R39): Added styles for the `#name-input` element to increase font size, enhancing the visibility of the input label.

beachlasagna commented:
good work but id like you to centre everything thats been aligned on the left like the reveal answer leaderboard and the icon of the game

khushiiagrawal commented:
> good work but id like you to centre everything thats been aligned on the left like the reveal answer leaderboard and the icon of the game

@beachlasagna 
### Done with all the changes as mentioned 

### HTML Changes:
* Added a new `div` to wrap the game title and logo for better layout control.
* Introduced a new `div` with `class="hidden2"` to wrap the "Reveal Answer" button and the score paragraph, which can be toggled for better user experience.

### CSS Changes:
* Introduced a `.hidden2` class to center-align elements within the new `div` added in the HTML. (`public/styles.css`, [public/styles.cssR73-R76](diffhunk://#diff-c0957aceb7c1368cd9919f48a72240ae31b22112283b1bd4d38c66a55ed584b9R73-R76))
* Updated the `#logo` style to increase the maximum width, center the logo, and adjust the margin for a more balanced layout. (`public/styles.css`, [public/styles.cssR113-R122](diffhunk://#diff-c0957aceb7c1368cd9919f48a72240ae31b22112283b1bd4d38c66a55ed584b9R113-R122))
